src/react-components/invite-dialog.js

```diff
@@ -89,9 +85,9 @@ export default class InviteDialog extends Component {
             )}
           {this.props.allowShare &&
             !navigator.share && (
-              <a href={tweetLink} className={styles.linkButton} target="_blank" rel="noopener noreferrer">
+              <button className={styles.linkButton} onClick={this.shareClicked.bind(this, shortLink)}>
```

Comment:
I would have written this to be `onClick=() => this.shareClicked(shortLink)` but this is probably OK -- i find i don't use `bind` anymore due to the property syntax. (currying the second arg is interesting tho, very FP :))